onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationUploadWorker.java

```diff
@@ -61,11 +87,31 @@ public void onFailure(@NonNull Exception exception) {
                     @Override
                     public void onSuccess(UploadTask.TaskSnapshot taskSnapshot) {
                         Log.d(TAG, logFileName + " uploaded successful");
+                        String userResponse = taskSnapshot.getMetadata().getCustomMetadata("Response");
+                        String feedbackText = taskSnapshot.getMetadata().getCustomMetadata("FeedbackText");
+                        String fileURL =  taskSnapshot.getStorage().getDownloadUrl().toString();
+                        Log.d(TAG, "Response - " + userResponse);
+                        Log.d(TAG, "FeedbackText - " + feedbackText);
+                        Log.d(TAG, "Download URL - " + fileURL);
+                        logFeedback(feedbackText, userResponse, fileURL);
                         boolean deleted = lFile.delete();
                         Log.v(TAG, logFileName + " deleted : " + deleted);
                     }
                 });
             }
         }
     }
+
+    private void logFeedback(String feedbackText, String userResponse, String fileName) {
+        Boolean wasGoodReminder;
+        if (userResponse.equals("yes")) {
```

Comment:
Could you please change this hard-coded string to a constant, likely in `donottranslate.xml`?

Reply:
Added hard-coded string to a constant in `donottranslate.xml`.